src/intl/en.json

```diff
@@ -155,5 +155,113 @@
+  "page-what-is-ethereum-native-title": "ETH",
+  "page-what-is-ethereum-native-alt": "The symbol for Ether (ETH)",
+  "page-what-is-ethereum-native-crypto": "Ethereum's native cryptocurrency and equivalent to Bitcoin. You can use ETH on Ethereum applications or for sending value to friends and family.",
+  "page-what-is-ethereum-wallets": "Wallets",
+  "page-what-is-ethereum-native-img-alt": "An illustration of a robot with a safe for a torso, used to represent Ethereum wallets",
+  "page-what-is-ethereum-wallets-desc": "How you manage your ETH and your Ethereum account. You'll need a wallet to get started – we'll help you choose one.",
+  "page-what-is-ethereum-dapps-title": "Ethereum dapps",
+  "page-what-is-ethereum-dapps-img-alt": "An illustration of a doge using an Ethereum application on a computer",
+  "page-what-is-ethereum-dapps-desc": "Products and services that run on Ethereum. There are dapps for finance, work, social media, gaming and more – meet the apps for our digital future.",
+  "page-get-eth-meta-title": "How to buy ETH",
+  "page-get-eth-meta-description": "How to buy ETH based on where you live and advice on how to look after it.",
+  "page-get-eth-where-to-buy-title": "Where to buy ETH",
+  "page-get-eth-where-to-buy-desc": "You can buy ETH from exchanges or from wallets directly.",
+  "page-get-eth-where-to-buy-desc-2": " Check which services you can use based on where you live.",
+  "page-get-eth-search-by-country": "Search by country",
+  "page-get-eth-CEX": "Centralized exchanges",
+  "page-get-eth-CEX-desc": "Exchanges are businesses that let you buy crypto using traditional currencies. They have custody over any ETH you buy until you send it to a wallet you control.",
+  "page-get-eth-DEX": "Decentralized exchanges (DEXs)",
+  "page-get-eth-DEX-desc": "If you want more control, buy ETH peer-to-peer. With a DEX you can trade without giving control of your funds to a centralized company.",
+  "page-get-eth-Try-Dex": "Try a Dex",
+  "page-get-eth-wallets": "Wallets",
+  "page-get-eth-wallets-purchasing": "Some wallets let you buy crypto with a debit/credit card, bank transfer or even Apple Pay. Geographical restrictions apply.",
+  "page-get-eth-wallets-link": "More on wallets",
+  "page-get-eth-disclaimer": "All exchanges, wallets and DEXs listed on this page are not official endorsements, and are provided for informational purposes only. We add products to this page based on criteria in our",
+  "page-get-eth-dapps-link": "dapps",
+  "page-get-eth-and": "and",
+  "page-get-eth-other-link": "exchanges",
+  "page-get-eth-policies": "listing policies. If you want to add an exchange or provide feedback on the policy",
+  "page-get-eth-raise-issue-link": "raise an issue in GitHub",
+  "page-get-eth-.": ".",
+  "page-get-eth-new-to-eth": " New to ETH? Here's an overview to page-get you started.",
```

Comment:
This is incorrect 🤔

Reply:
Should be: `New to ETH? Here's an overview to get you started.`. [You can see it in production](https://ethereum.org/en/get-eth/):
![Image 2020-09-09 at 1 14 09 PM](https://user-images.githubusercontent.com/8097623/92648596-82b6e400-f29e-11ea-8259-2d282f0cab80.png)